pkg/controller/handlers/accesscontrolrule/helper.go

```diff
@@ -0,0 +1,109 @@
+package accesscontrolrule
+
+import (
+	"fmt"
+
+	v1 "github.com/obot-platform/obot/pkg/storage/apis/obot.obot.ai/v1"
+	"github.com/obot-platform/obot/pkg/system"
+	gocache "k8s.io/client-go/tools/cache"
+)
+
+type Helper struct {
+	acrInformer gocache.SharedIndexInformer
```

Comment:
nit: looks like we always call `GetIndexer()` on this. Can this just be the indexer instead of the informer?

Reply:
Yep

Reply:
Fixed